generators/entity-client/templates/react/src/test/javascript/spec/app/entities/entity-reducer.spec.ts.ejs

```diff
@@ -25,7 +25,7 @@ import { expect } from 'chai';
 import reducer, {
   ACTION_TYPES, createEntity, deleteEntity, getEntities,
   getEntity, updateEntity
-} from 'app/entities/<%= entityFileName %>/<%= entityFileName %>.reducer';
+} from 'app/entities<% if (applicationType === 'gateway' && locals.microserviceName) { %>/<%= microserviceName.toLowerCase() %><% } %>/<%= entityFileName %>/<%= entityFileName %>.reducer';
```

Comment:
this should be applied only when the generated entity is grouped by service, similar to how its done in angular

Reply:
I know, this wasn't meant to be merged yet.

Reply:
ha ha, I guess @jdubois is on a mergy mood today :)